### DISCUSSION TITLE
Webcam's einbinden in v3

### DISCUSSION THREAD

**Discussion by robert-ke** [Zefau/ioBroker.jarvis#980] [General]
Hallo zusammen,

ich habe nun auch auf v3.0.0-alpha.70 aktualisiert. Soweit alles sehr schick und läuft alles perfekt! Leider sind Webcams die per JPG Link integriert waren nicht mehr funktionsfähig. Ein Blick in die Widgets zum DisplayImage zeigt mir, dass die Links weg sind:

![image](https://user-images.githubusercontent.com/74955542/132954466-8a7bd0a2-9cdb-4c1c-a007-61dc958ab1e2.png)

Irgendwie fehlt auch das Feld wo man den Link einfügen kann - oder finde ich nur den "neuen" Weg nicht zum Ziel?

Vielen Dank!

**Comment by Zefau** [comment]
Link input kommt noch (https://github.com/Zefau/ioBroker.jarvis/issues/945). Übergang erstmal via State.
